### PR TITLE
Switch from bun to pnpm

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: apps/mobile-wallet/src/components/inputs/Input.tsx [hunk 1]
```diff
@@ -126,6 +126,7 @@ const Input = <T extends InputValue>({
           onFocus={handleFocus}
           onBlur={handleBlur}
           ref={usedInputRef}
+          forwardedAs={TextInput}
```
Comment (nop33) [inline apps/mobile-wallet/src/components/inputs/Input.tsx:129]: Need to test this

File: apps/mobile-wallet/src/components/inputs/ColorPicker.tsx [hunk 2]
```diff
@@ -81,8 +81,10 @@ const Colors = styled.View`
   height: 100%;
 `
 
-const Color = memo(styled(HighlightRow)<{ color: string }>`
+const Color = memo(styled.Pressable<{ color: string }>`
```
Comment (nop33) [inline apps/mobile-wallet/src/components/inputs/ColorPicker.tsx:84]: Need to test this

File: apps/explorer/src/components/Table/Table.tsx [hunk 3]
```diff
@@ -38,7 +38,7 @@ interface TableProps {
 
 export interface TDStyle {
   tdPos: number
-  style: FlattenInterpolation<ThemeProps<DefaultTheme>>
+  style: ReturnType<typeof css>
```
Comment (nop33) [inline apps/explorer/src/components/Table/Table.tsx:41]: Need to test this

File: apps/mobile-wallet/src/components/inputs/ColorPicker.tsx (hunk 2 above)
Comment (nop33) [reply apps/mobile-wallet/src/components/inputs/ColorPicker.tsx:84]: It works

File: apps/explorer/src/components/Table/Table.tsx (hunk 3 above)
Comment (nop33) [reply apps/explorer/src/components/Table/Table.tsx:41]: works

File: apps/mobile-wallet/src/components/inputs/Input.tsx (hunk 1 above)
Comment (nop33) [reply apps/mobile-wallet/src/components/inputs/Input.tsx:129]: works

**Comment by nop33** [comment]
Some good testing will be needed on this. There were many brainfuck moments.

**Review by mvaivre — APPROVED** [review]
👏 👏 👏

**Pull request merged**